docs/run-platform-server/prerequisites.mdx

```diff
@@ -3,56 +3,45 @@ title: Prerequisites
 sidebar_position: 10
 ---
 
-Horizon only has one true dependency: a PostgreSQL server that it uses to store data that has been processed and ingested from Stellar Core. **Horizon requires PostgreSQL version >= 9.5**.
+The Horizon service is responsible for synchronizing with the Stellar network and processing ledger data. To understand the scope of Horizon's services, please read the [configuring](./configuring.mdx) section before you move on to the prerequisites for computation.
 
-As far as system requirements go, there are a few main things to keep in mind. Starting from version 2.0, Horizon must be run as a standalone service. A full Horizon build consists of three functions:
+The Horizon service can be [installed](./installing.mdx) on bare metal or a virtual machine. It is natively supported on both Linux and Windows operating systems.
 
-1. **ingesting data** _from_ the decentralized Stellar network,
-1. **submitting transactions** _to_ the network, and
-1. **serving** API requests
+**Single Instance Deployment model**
 
-The first two happen through _Captive Core_, a pared down, non-validating version of Stellar Core packaged directly into Horizon.
+For a basic setup using the [Single Instance Deployment model](./configuring.mdx#single-instance-deployment), you will need two distinct compute instances:
```

Comment:
```suggestion
For a basic setup using the [Single Instance Deployment model](./configuring.mdx#single-instance-deployment), you will need a sum of two distinct compute profiles:
```

used 'compute profile', as the horizon and db could be on the same host o/s, as long as its resources = sum of both mentioned here correct?

Reply:
Yes, I suppose.